### PR TITLE
Add Cauldron

### PR DESCRIPTION
**Pull request by Creeperface01**
its able to merge now so i hope you can merge it before it wont be able to merge :D

### CODE REVIEW

**Comment by NycuRO** [comment]
@Creeperface01 Epic 👍

**Comment by NycuRO** [comment]
@Creeperface01 But problem is plugins. Not exist more plugins for use all nukkit..

File: src/main/java/cn/nukkit/block/BlockCauldron.java [hunk 1]
```diff
@@ -0,0 +1,223 @@
...
+
+        new BlockEntityCauldron(this.level.getChunk((int) this.x >> 4, (int) this.z >> 4), nbt);
+        this.getLevel().setBlock(block, this, true, true);
+        return true;
+    }
+
+    @Override
+    public int[][] getDrops(Item item) {
+        if (item.getTier() >= ItemTool.TIER_WOODEN) {
+            return new int[][]{{}};
```
Comment (Pub4Game) [inline src/main/java/cn/nukkit/block/BlockCauldron.java]: It's wrong, see drop in other blocks

Comment (Creeperface01) [inline src/main/java/cn/nukkit/block/BlockCauldron.java]: yes thx

**Comment by Creeperface01** [comment]
sure why do not make 2 commits instead of 1........ -,-

File: src/main/java/cn/nukkit/block/BlockCauldron.java [hunk 2]
```diff
@@ -0,0 +1,223 @@
...
+
+        new BlockEntityCauldron(this.level.getChunk((int) this.x >> 4, (int) this.z >> 4), nbt);
+        this.getLevel().setBlock(block, this, true, true);
+        return true;
+    }
+
+    @Override
+    public int[][] getDrops(Item item) {
+        if (item.getTier() >= ItemTool.TIER_WOODEN) {
+            return new int[0][0];
```
Comment (Pub4Game) [inline src/main/java/cn/nukkit/block/BlockCauldron.java]: No drop? 3:

Comment (Creeperface01) [inline src/main/java/cn/nukkit/block/BlockCauldron.java]: OMG :D sry thx

**Comment by Creeperface01** [comment]
Can i try to implement some mobs from pureEntities?

**Comment by Snake1999** [comment]
@Creeperface01 Of course you can! We will be happy after entities implemented :)

**Comment by NycuRO** [comment]
@Snake1999 I can help and I? :)

**Comment by MrPowerGamerBR** [comment]
@Creeperface01 if you are going to implement, please make blazes/ghasts/etc only spawn in NETHER.

Because... You know, I think nobody would like having blazes flying around in the server spawn, right?

**Comment by Creeperface01** [comment]
I think ill add AI first, but spawning is not good in pureentities

**Comment by MrPowerGamerBR** [comment]
Also add a disable AI option in Nukkit.yml

Em 5 de out de 2016 8:17 AM, "Creeperface01" <notifications@github.com>
escreveu:

**Comment by Creeperface01** [comment]
Ok

**Comment by kniffo80** [comment]
@Creeperface01 if i can, i would volunteer implementing Mob AI. Had a deeper look into PureEntities and forked that already. But i also like the idea having Mob AI directly in Nukkit.

**Comment by MrPowerGamerBR** [comment]
Now everyone decided to create a pull request to port PureEntities to Nukkit source code. 😋

**Comment by Creeperface01** [comment]
@Muqsit add it please :D

**Pull request merged**